packages/svelte/test/hydration/samples/repairs-apparent-static-content/main.svelte

```diff
@@ -0,0 +1 @@
+<h1>Hello {globalThis.whatever}!</h1>
```

Comment:
I don't think that this test is correct / is testing what's actually happening. What we need is to tell the test that there are two variants: `<h1>Hello {'server'}</h1>` and `<h1>Hello {'client'}</h1>`. Having written that, maybe that's possible to do using some file name conventions and sprucing up the test suite? mhm ..